apps/win32/package.json

```diff
@@ -31,23 +31,22 @@
     "react-native-svg": "^11.0.0"
   },
   "devDependencies": {
-    "@office-iss/react-native-win32": "0.27.3",
-    "@office-iss/rex-win32": "0.0.0-devmain.12823.10000",
+    "@office-iss/react-native-win32": "0.62.0-preview.3",
```

Comment:
FYI we're on preview 4 now

Reply:
Currently this is set to match the rex-win32 version that we are pulling in.  Once we get the publishing pipeline well established on the native side I'd like to figure out how to establish regular version bumping in this repo.